pyuvsim/tests/test_analyticbeam.py

```diff
@@ -202,16 +198,19 @@ def test_gaussbeam_values():
 
     engine = pyuvsim.UVEngine(task)
     engine.apply_beam()
-    altitudes = task.sources.alt_az[0]   # In radians.
-    coherencies = np.real(engine.apparent_coherency[0, 0] + engine.apparent_coherency[1, 1]).astype(float)  # All four components should be identical
+    altitudes = task.sources.alt_az[0]  # In radians.
+    coherencies = np.real(
+        engine.apparent_coherency[0, 0] + engine.apparent_coherency[1, 1]
+    ).astype(float)  # All four components should be identical
```

Comment:
I think this split is caused by the comment. Let's move the comment to before the line to avoid it.

Reply:
I put it on the line before, but I still have to split the line.